feat: add min max size for random slice map And allowing one element for Oneof

autumn31 opened the pull request:
wish to add a feature that can set minimum random size for slice and map

bxcodec commented:
Hi @autumn31  can you help to check the test, it's failing on the CI

Sorry for the late reply, been busy lately. 🙇🏻

autumn31 commented:
Test fixed. Also I updated the mr title, since I added another feature that `Oneof should accept "one element" cases`.
Wonder how you think about this change?

bxcodec commented on faker.go:
```diff
@@ -1087,6 +1090,26 @@ func extractNumberFromText(text string) (int, error) {
+	if strings.Contains(argsList[0], ",,") {
+		return nil, fmt.Errorf(ErrDuplicateSeparator)
+	}
+	fmt.Printf("argsList(%v)\n", argsList[0])
```
can remove this?

autumn31 replied:
Sure, this line was removed in the latest commit.

bxcodec reviewed (commented):
Hey @autumn31 

So sorry for the late reply. 
Minor comment, otherwise LGTM

bxcodec reviewed (approved):
LGTM

bxcodec commented:
Hey @autumn31 , 

Thanks for the PR, I'll release it soon. In the mean time, you can try to test on your project using the master branch for the source
